.github/workflows/macstadium-e2e.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         run:  yarn audit-ci --moderate --config audit-ci.json
 
       - name: Lint
-        run: yarn lint
+        run: yarn format:check && yarn lint
```

Comment:
this should be part of `yarn lint` since it actually lints for mistakes

Reply:
Yeah good callout. I have [another PR open](https://github.com/rainbow-me/rainbow/pull/4070) to quiet logs in CI, which requires separate scripts. In general I think composing these scripts together is better than having one big script. Thoughts?

Reply:
and yarn lint checks for both ts and js linter errors
it used to be part of eslint, and people would call `yarn lint` locally to see what was falling

Reply:
Fair point. Done!